pages/authentification.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import imaplib
+import email
+
+import time
+from tempmail import TempMail
+from selenium.webdriver.common.by import By
+
+from pages.base import Base
+
+
+class Authentification(Base):
+
+    _login_with_email_button_locator = (By.CSS_SELECTOR, '.auth0-lock-passwordless-button.auth0-lock-passwordless-big-button')
+    _email_input = (By.CSS_SELECTOR, '.auth0-lock-passwordless-pane>div>div>input')
+    _send_email_button_locator = (By.CSS_SELECTOR, '.auth0-lock-passwordless-submit')
+    _email_sent_successful_message_locator = (By.CSS_SELECTOR, '.auth0-lock-passwordless-confirmation>p')
+
+
+    def sign_in(self, email):
+        self.wait_for_element_visible(*self._login_with_email_button_locator)
+        self.selenium.find_element(*self._login_with_email_button_locator).click()
+        self.wait_for_element_visible(*self._email_input)
+        self.selenium.find_element(*self._email_input).send_keys(email)
+        self.selenium.find_element(*self._send_email_button_locator).click()
+        # Wait for the email to be received 5 seconds after login
+        time.sleep(5)
```

Comment:
We should have a wait, not a sleep. In some cases 5 seconds might not be enough, and in a lot of cases it may be too much.